source/new.rst

```diff
@@ -124,26 +146,37 @@ messages.
 
 The protocol follows a single simple UI workflow:
 A peer "shows" bootstrap data
-that is then "read" by the other peer through a trusted (out-of-band) channel.
+that is then "read" by the other peer through a second channel.
 This means that,
 as opposed to current fingerprint verification workflows,
 the protocol only runs once instead of twice,
 yet results in the two peers having verified keys of each other.
 
-On mobile phones, trusted channels are typically implemented using QR codes,
+Between mobile phones,
+showing and scanning a QR code
+constitutes a second channel,
 but transferring data via USB, Bluetooth, WLAN channels or phone calls
 is possible as well.
+
 Recall that
 we assume that
-our active attacker *cannot* observe or modify data transferred via the
-trusted channel.
+our active attacker *cannot* observe or modify data transferred
+via the second channel.
+
+An attacker who can alter messages
+but has no way of reading or manipulating the second channel
+can disrupt the verification flow.
```

Comment:
suggest to be more explicit: "Am attack who can alter messages on the first channel but has no way of reading or manipulating the second channel can prevent the verification protocol from completing successfully."